mhep/mhep/v1/templates/v1/view.html

```diff
@@ -778,7 +778,7 @@ <h3>This scenario is locked!</h3>
         sidebar_visible = true;
         $("#sidebar").css("left", "340px");
         if (width >= max_wrapper_width)
-            $("#wrapper").css("padding-left", "330px");
```

Comment:
I think this change meant to be made on `dev`, not `v1`?

Reply:
Ah yes.  I originally modified the v1 app (since there was nothing else) but missed this file when moving it all across to dev